reviewdog parse error in Vue 3: failed to unmarshal rdjson (DiagnosticResult)

**System Information:** 
```
macOS Monterey 
Version 12.6
Macbook Pro (M1, 2020)
Chip Apple M1
```

**Software / Frameworks:**
```
Vue.js 3 with TypeScript
Node.js version v18.14.0

```

**reviewdog.yml**
```
name: reviewdog
on: [pull_request]
jobs:
  eslint:
    name: runner / eslint
    runs-on: ubuntu-latest
    steps:
      - name: Check out code.
        uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '14'
          cache: 'npm'
      - name: Install dependencies
        run: npm install
      - name: eslint
        uses: reviewdog/action-eslint@v1
        with:
          github_token: ${{ secrets.github_token }}
          reporter: github-pr-review
          eslint_flags: '-c .eslintrc.js src'

```

I am using the above `reviewdog.yml` file on Github Actions and `action-eslint@v1` keeps failing today with the following error:
```
Run reviewdog/action-eslint@v1
  with:
    github_token: ***
    reporter: github-pr-review
    eslint_flags: -c .eslintrc.js src
    level: error
    filter_mode: added
    fail_on_error: false
    workdir: .
    tool_name: eslint
Run $GITHUB_ACTION_PATH/script.sh
  $GITHUB_ACTION_PATH/script.sh
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    REVIEWDOG_VERSION: v0.1[4](https://github.com/xxx/actions/runs/xx021642/jobs/xxx#step:5:4).1
    INPUT_GITHUB_TOKEN: ***
    INPUT_LEVEL: error
    INPUT_REPORTER: github-pr-review
    INPUT_FILTER_MODE: added
    INPUT_FAIL_ON_ERROR: false
    INPUT_REVIEWDOG_FLAGS: 
    INPUT_ESLINT_FLAGS: -c .eslintrc.js src
    INPUT_WORKDIR: .
    INPUT_TOOL_NAME: eslint
    NODE_OPTIONS: 
🐶 Installing reviewdog ... https://github.com/reviewdog/reviewdog
[6](https://github.com/xxx/actions/runs/xxx/jobs/xxx#step:5:6).14.1[8](https://github.com/YPBConnectPlatform/connect-next-frontend/actions/runs/4249021642/jobs/7388780998#step:5:8)
eslint version:6.[14](https://github.com/xxx/actions/runs/xxx/jobs/xxx#step:5:15).[18](https://github.com/xxx/actions/runs/4249021642/jobs/xxxss#step:5:19)
 Running eslint with reviewdog 🐶 ...
  /bin/sh: 1: .eslintrc.js: not found
  reviewdog: parse error: failed to unmarshal rdjson (DiagnosticResult): proto: syntax error (line 1:1): unexpected token 
Error: Process completed with exit code 1.
```
I already have an .eslintrc.js file in the root of Vue 3 project. `reviewdog` was working fine but this issue just happened today. 
Is there any fix for this issue?

--- a/.github/workflows/reviewdog.yml
+++ b/.github/workflows/reviewdog.yml
@@ -6,6 +6,7 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
       matrix:
         node_version: [14, 16, 18]
 

--- a/.github/workflows/reviewdog.yml
+++ b/.github/workflows/reviewdog.yml
@@ -4,8 +4,17 @@ jobs:
   eslint:
     name: runner / eslint
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node_version: [14, 16, 18]
+
     steps:
       - uses: actions/checkout@v3
+      - name: setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node_version }}
       - name: eslint-github-pr-check
         uses: ./
         with:

--- a/script.sh
+++ b/script.sh
@@ -11,7 +11,7 @@ echo '::group::🐶 Installing reviewdog ... https://github.com/reviewdog/review
 curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b "${TEMP_PATH}" "${REVIEWDOG_VERSION}" 2>&1
 echo '::endgroup::'
 
-npx --offline eslint --version
+npx --no-install -c 'eslint --version'
 if [ $? -ne 0 ]; then
   echo '::group:: Running `npm install` to install eslint ...'
   npm install
@@ -21,7 +21,7 @@ fi
 echo "eslint version:$(npx --offline eslint --version)"
 
 echo '::group:: Running eslint with reviewdog 🐶 ...'
-npx --offline eslint -f="${ESLINT_FORMATTER}" ${INPUT_ESLINT_FLAGS:-'.'} \
+npx --no-install -c "eslint -f="${ESLINT_FORMATTER}" ${INPUT_ESLINT_FLAGS:-'.'}" \
   | reviewdog -f=rdjson \
       -name="${INPUT_TOOL_NAME}" \
       -reporter="${INPUT_REPORTER:-github-pr-review}" \

